code/modules/power/cell.dm

```diff
@@ -36,7 +36,8 @@
 	START_PROCESSING(SSobj, src)
 	charge = !isnull(starting_charge) ? starting_charge : maxcharge
 	if(ratingdesc)
-		desc += " This one has a power rating of [DisplayPower(maxcharge)], and you should not swallow it."
+		//state of charge is in KJ so we multiply it by 1000 to get Joules
+		desc += " This one has a power rating of [DisplayJoules(maxcharge * 1000)], and you should not swallow it."
```

Comment:
Personally I would actually express cell charge in kJ rather than J. This is _strictly_ a personal preference thing, however.

Reply:
DisplayJoules should give the correct suffix. It's going to be the same as the old description but with joules instead of watts and not off by a factor of 1000